client/client-sdk-javascript/src/orbs-contract-adapter.ts

```diff
@@ -1,15 +1,22 @@
 import { OrbsClient } from "./orbs-client";
 import { SendTransactionOutput } from "orbs-interfaces";
+import { Address } from "./address";
+import * as crypto from "crypto";
 
 export type OrbsContractMethodArgs = [string | number] | any[];
 
 export class OrbsContractAdapter {
```

Comment:
consider renaming this class, as `Adapter` is a mechanical name that does not add any further information. Maybe `OrbsContractClient`? or simply `OrbsContract`?